src/App.tsx

```diff
@@ -1,23 +1,70 @@
 import styles from "./App.module.css";
+import FlightTable from "./components/FlightTable/FlightTable";
+import { faker } from "@faker-js/faker";
+import React from "react";
+import Button from "@mui/material/Button";
 
 export interface Flight {
   // ... fields here
+  flightNumber: string;
+  origin: string;
+  destination: string;
+  departureTime: string;
+  arrivalTime: string;
+  flightDuration: number;
 }
 
 const App = () => {
   // state for flights should go here
+  const [flightList, setFlightList] = React.useState<Flight[]>([]);
+
+  function formatAMPM(date: Date): string {
+    let hours = date.getHours();
+    let minutes = String(date.getMinutes());
+    const ampm = hours >= 12 ? "pm" : "am";
+    hours = hours % 12;
+    hours = hours ? hours : 12; // the hour '0' should be '12'
+    minutes = Number(minutes) < 10 ? "0" + minutes : minutes;
+    const strTime = hours + ":" + minutes + " " + ampm;
+    return strTime;
+  }
+
+  function generateFlight(): Flight {
+    const newFlight: Flight = {
+      flightNumber: faker.airline.flightNumber(),
+      origin: String(faker.airline.airport().iataCode),
+      destination: String(faker.airline.airport().iataCode),
+      departureTime: formatAMPM(faker.date.anytime()),
+      arrivalTime: formatAMPM(faker.date.anytime()),
```

Comment:
Love how you pass the faker function into the formatFunction that you wrote, this is great style!